packages/common/src/controllers/AccountController.ts

```diff
@@ -550,9 +530,9 @@ export default class AccountController {
       selectingProfileAvatar: null,
     });
 
-    this.profileController?.unpersistProfile();
+    this.profileController.unpersistProfile();
 
-    await this.favoritesController?.restoreFavorites();
-    await this.watchHistoryController?.restoreWatchHistory();
+    await this.favoritesController.restoreFavorites().catch(logDev);
```

Comment:
Is there anywhere where we want these methods to throw errors? If not, does it make sense to add try/catch inside of the restoreFavorites and restoreWatchHistory implementations instead of catching on each call?

Reply:
We had a single case where this threw an error (duplicate serie IDs). The controller in concert should indeed decide whether errors are thrown or not. I will update the `restoreFavorites` and `restoreWatchHistory` methods instead.